src/bento/userLoginData.js

```diff
@@ -0,0 +1,38 @@
+import Google from '../assets/login/google.png';
+import LoginGov from '../assets/login/login.gov.png';
+import NihItrust from '../assets/login/nih_itrust.png';
+import { registrationRoute } from './siteWideConfig';
+
+export const pageTitle = 'Login or Register';
+
+export const loginProvidersData = {
+  google: {
+    key: 'google',
+    icon: Google,
```

Comment:
can we pull in logo icons from datacommons assets

Reply:
Will make that change.